drivers/video/fb.c

```diff
@@ -1505,15 +1492,14 @@ int fb_register(int display, int plane)
 
   nxmutex_init(&fb->lock);
 
-  ret = register_driver(devname, &g_fb_fops, 0666, (FAR void *)fb);
+  ret = register_driver(devname, &g_fb_fops, 0666, fb);
```

Comment:
Can we change `snprintf(devname, 16` to `snprintf(devname, sizeof(devname)` few lines above?

Reply:
Anyway can be processed in separate PR